src/SimpleSAML/Command/UpdateTranslatableStringsCommand.php

```diff
@@ -31,6 +31,22 @@
 use function ksort;
 use function sprintf;
 
+
+function cloneIteratorToTranslations($ret, $iterator)
```

Comment:
This should probably be private method inside the class.. The parameters should be type-hinted.

Reply:
Agreed. I did mention that this was a quick and dirty PoC :)

Reply:
You did ;)  We're strict like this! :goberserk:

Reply:
Updated, https://github.com/simplesamlphp/simplesamlphp/pull/1961/commits/9a09e5e89415fa8d56525e0b1c408f9582956c59